java-util/src/main/java/io/druid/java/util/common/io/smoosh/Metadata.java

```diff
@@ -21,7 +21,7 @@
 
 /**
 */
-class Metadata
+public class Metadata
```

Comment:
yes, the customIndexMerger may need to use Metadata, like in my customIndexMerger implement.